pkgs/development/tools/misc/distcc/default.nix

```diff
@@ -27,10 +27,10 @@ let
                             CXXFLAGS="-O2 -fno-strict-aliasing"
           --mandir=$out/share/man
                             ${if sysconfDir == "" then "" else "--sysconfdir=${sysconfDir}"}
-                            ${if static then "LDFLAGS=-static" else ""}
-                            --with${if static == true || popt == null then "" else "out"}-included-popt
-                            --with${if avahi != null then "" else "out"}-avahi
-                            --with${if gtk3 != null then "" else "out"}-gtk
+                            ${lib.optionalString static "LDFLAGS=-static"}
+                            --with${lib.optionalString (static != true && popt != null) "out"}-included-popt
+                            --with${lib.optionalString (avahi == null) "out"}-avahi
+                            --with${lib.optionalString (gtk3 == null) "out"}-gtk
```

Comment:
This is better done with `lib.withFeature`

Reply:
thanks did not know this. is fixed